boxsdk/client/client.py

```diff
@@ -317,6 +317,39 @@ def create_group(self, name):
         )
 
     @api_call
+    def get_recent_items(self, limit=100, offset=0, fields=None):
+        """
+        Get the user's recently accessed items.
+
+        :param: limit
+            The maximum number of items to return
+        :type: limit
+            `int`
+        :param offset:
+            The index at which to start returning items.
+        :type offset:
+            `int`
+        :param fields:
+            List of fields to request.
```

Comment:
Actually, these do apply to the underlying file or folder. I'll mention that. The recent_item type always has the same fields, but returns the 'mini' file/folder format by default, so this is to request more fields. It's similarly unclear in the API documentation, but that is what it does in practice. I'll update this to be more clear.